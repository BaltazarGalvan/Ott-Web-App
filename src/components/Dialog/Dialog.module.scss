@use '../../styles/variables';
@use '../../styles/theme';

.dialog {
  max-width: 450px;
  padding: 24px;
  color: theme.$modal-color;
  background-color: theme.$modal-bg;
  border-radius: 6px;
}
