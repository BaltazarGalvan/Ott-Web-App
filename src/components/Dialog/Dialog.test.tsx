import React from 'react';
import { render } from '@testing-library/react';

import Dialog from './Dialog';

describe('<Dialog>', () => {
  test('renders and matches snapshot', () => {
    const { container } = render(<Dialog />);

    expect(container).toMatchSnapshot();
  });
});
