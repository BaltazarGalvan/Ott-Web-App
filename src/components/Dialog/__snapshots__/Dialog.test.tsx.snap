// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Dialog> renders and matches snapshot 1`] = `<div />`;
